lib/Migration/Version011000Date20210206170027.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OCA\Files_Retention\Migration;
+
+use Closure;
+use OCP\DB\ISchemaWrapper;
+use OCP\Migration\IOutput;
+use OCP\Migration\SimpleMigrationStep;
+
+/**
+ * Auto-generated migration step: Please modify to your needs!
+ */
+class Version011000Date20210206170027 extends SimpleMigrationStep {
+
+	/**
+	 * @param IOutput $output
+	 * @param Closure $schemaClosure The `\Closure` returns a `ISchemaWrapper`
+	 * @param array $options
+	 * @return null|ISchemaWrapper
+	 */
+	public function changeSchema(IOutput $output, Closure $schemaClosure, array $options): ?ISchemaWrapper {
+		/** @var ISchemaWrapper $schema */
+		$schema = $schemaClosure();
+
+		if ($schema->hasTable('retention')) {
+			$table = $schema->getTable('retention');
+
+			$table->addColumn('time_after', 'integer', [
+				'notnull' => true,
```

Comment:
@nickvergessen does this work with Oracle?

Reply:
No because the default is 0 and 0 on int is "null"

Reply:
As I thought.

@sualko can you make it nullable. And add the logic that if it is null you just assume 0?
(else the oracle DB fails hard).

Reply:
Since I use a `(int)` cast everywhere, that should be no issue.

Reply:
But then it should also be nullable. Else oracle complains (0 === null in oracles twisted world)

Reply:
yeah thats what I understood, when default is 0 you need to be nullable

Reply:
Seams to be fun to be working with Oracle. I will change that asap.

Reply:
You have no idea :P